emission/core/wrapper/user.py

```diff
@@ -127,10 +127,14 @@ def getSettings(self):
 
   @staticmethod
   def createProfile(uuid, ts):
-    initProfileObj = {'user_id': uuid,
+    initProfileObj = {
+                      'user_id': uuid,
                       'source':'Shankari',
```

Comment:
nit: can eventually remove this `source` since it is hardcoded and never used. I don't remember the full history (will need to check the blame) but IIRC, it was an early attempt at multi-tenancy (e.g. users from my team versus from other teams at Berkeley)